app/src/main/res/layout/item_campaign.xml

```diff
@@ -7,11 +7,23 @@
     android:layout_width="wrap_content"
     android:layout_height="@dimen/campaign_height">
 
-    <ImageView
+    <FrameLayout
         android:layout_width="match_parent"
-        android:id="@+id/image"
-        android:scaleType="centerCrop"
-        android:layout_height="match_parent"/>
+        android:id="@+id/image_container"
+        android:layout_height="@dimen/campaign_height">
+
+        <ImageView
+            android:id="@+id/image"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/campaign_height"
+            android:scaleType="centerCrop"
+            android:src="@drawable/placeholder"/>
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@drawable/darken_30"/>
+    </FrameLayout>
 
     <org.dosomething.letsdothis.ui.views.typeface.CustomTextView
```

Comment:
FIXME: Both the title and call to action should be centered. Right now only the title is centered.